File: openfinalfinal/u202120418/u202120418/src/main/java/org/hign/platform/u202120418/assessment/interfaces/rest/resources/MentalStateExamResource.java
package org.hign.platform.u202120418.assessment.interfaces.rest.resources;

/**
 * @author Fiorella Jarama Peñaloza
 * @version 1.0
 */
public record MentalStateExamResource(
        Long id,
        Long patientId,
        String examDate,
        Integer orientationScore,
        Integer attentionAndCalculationScore,
        Integer registrationScore,
        Integer recallScore,
        Integer languageScore,
        String examinerNationalProviderIdentifier
) {
}

File: web/HIGN/HIGN/Assessment/Interfaces/REST/Resources/MentalStateExamResource.cs
namespace HIGN.Assessment.Interfaces.REST.Resources;

public record MentalStateExamResource(long Id,long PatientId, string NationalProvideIdentifier, DateTime ExamDate, int OrientationScore, int RegistrationScore, int AttentionAndCalculationScore, int RecallScore, int LanguageScore, long ExaminerId);